docs/Modding/Tilesets/debugger.rst

```diff
@@ -0,0 +1,48 @@
+Tileset Debugger
+================
+
+.. versionadded:: 3.0-alpha6
+
+The Tileset Debugger, accessible from the :guilabel:`Game` menu, lets you
+inspect how the map is drawn. This is very helpful when developing a tileset, to
+understand why something is rendering incorrectly or to understand how other
+tilesets work.
+
+.. note::
```

Comment:
Do we specifically need this note? Its pretty obvious we like suggestions.

Reply:
I wanted to emphasize this because the direction it takes in its early days will directly affect the quality of the end result (unless we rewrite it three times)